integration-libs/opf/base/core/services/opf-endpoints.service.ts

```diff
@@ -7,20 +7,22 @@
 import { Injectable } from '@angular/core';
 import {
   BaseSiteService,
-  Config,
   DynamicAttributes,
   StringTemplate,
 } from '@spartacus/core';
 
+import { OpfApiConfig, OpfConfig } from '@spartacus/opf/base/root';
+
 @Injectable({
   providedIn: 'root',
 })
 export class OpfEndpointsService {
   private _activeBaseSite: string;
 
   constructor(
-    private config: Config,
-    private baseSiteService: BaseSiteService
+    protected opfConfig: OpfConfig,
+    protected opfApiConfig: OpfApiConfig,
```

Comment:
[MEDIUM]
Do we really need the config injected 2 times? Isn't it the same object in the end of the day?